tests/package.json

```diff
@@ -3,18 +3,18 @@
   "private": true,
   "scripts": {
     "test": "METEOR_PACKAGE_DIRS=../package TEST_BROWSER_DRIVER=puppeteer meteor test --once --driver-package meteortesting:mocha --raw-logs",
-    "test:watch": "METEOR_PACKAGE_DIRS=../package TEST_BROWSER_DRIVER=puppeteer TEST_WATCH=1 meteor test --driver-package meteortesting:mocha --raw-logs",
+    "test:watch": "METEOR_PACKAGE_DIRS=../package TEST_BROWSER_DRIVER=puppeteer TEST_WATCH=1 meteor test --driver-package meteortesting:mocha",
     "test:watch:browser": "METEOR_PACKAGE_DIRS=../package TEST_WATCH=1 meteor test --driver-package meteortesting:mocha --raw-logs"
   },
   "dependencies": {
-    "@babel/runtime": "^7.11.2",
+    "@babel/runtime": "^7.23.1",
     "babel-polyfill": "6.26.0",
     "jquery": "^3.5.1",
     "meteor-node-stubs": "1.0.1",
-    "simpl-schema": "1.10.2"
+    "simpl-schema": "3.4.1"
```

Comment:
This is breaking things, please revert back to v1.x

Reply:
There do not seem to be any breaking changes indicated. However, I am reverting to version 1.10.2 with the indicated vulnerabilities fixed by 1.12.1:

NOTE: To avoid potential ReDoS vulnerabilities, upgrade to this version and discontinue the use of the built-in regular expressions Url and Phone, which are deprecated and will be removed in a future major release. Instead, use the new URL() to validate URLs and a suitable phone number library to validate phone numbers.

Reply:
I think you can bump to the latest 1.x it's just that 3.x has dropped meteor support and messagebox so it will not break the package tests but may be breaking for ppl using this package with translated messages

Reply:
Consider that the package.json is only useful for testing purposes, it has no impact on the package. Anyway, I have updated the version to 1.12.3.